### PR TITLE
Create Invoice from `appuio-cloud-reporting`

### PR DESCRIPTION
**Pull request by bastjan**
## Summary

* Create Invoice from `appuio-cloud-reporting`.

## Checklist

- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [x] Update tests.

### CODE REVIEW

File: invoice/invoice.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	created, err := client.CreateInvoice(ctx, invoice)
+	if err != nil {
+		return created.ID, fmt.Errorf("error creating invoice in odoo: %w", err)
+	}
+
+	createdLines := make([]model.InvoiceLine, 0, len(lines))
+	for _, line := range lines {
+		line, err := client.InvoiceAddLine(ctx, created.ID, line)
+		if err != nil {
+			return created.ID, fmt.Errorf("error adding line to invoice %d: %w", created.ID, err)
```
Comment (glrf) [inline invoice/invoice.go:71]: Nitpick: It might be helpful when debugging to add the line id or name to the error

File: invoice/invoice.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+		}
+		createdLines = append(createdLines, line)
+	}
+	_ = createdLines
```
Comment (glrf) [inline invoice/invoice.go:75]: I suspect that is some debugging reasons that this is still around and you forgot to clean up?

**Review by glrf — APPROVED** [review]
LGTM :rocket:

File: invoice/invoice.go (hunk 2 above)
Comment (bastjan) [reply invoice/invoice.go:75]: Uh yes that was an oversight 😅

**Pull request merged**